### DISCUSSION TITLE
I dont know how to get type of dependencies when switching to Pnp

### DISCUSSION THREAD

**Discussion by nhattran998** [yarnpkg/berry#3810] [Q&A] [answered]
I start to open sources a basic monorepo. But When I decided to try the Pnp instead of `node_modules`. I cannot get the type of dependencies anymore!

Here is the repo: https://github.com/techmely/techmely

I read the docs, but can not find some information to do that... Hope anyone can help me solve it!  Thanks <3

**Comment by merceyz** [comment, marked as answer]
See https://github.com/yarnpkg/berry/issues/3722 for information and https://github.com/yarnpkg/berry/issues/3722#issuecomment-979992345 for the solution

**Reply by nhattran998** [reply]
Thank you for your comment <3! I really love this tools
